### PR TITLE
Translate DexiNet model from PyTorch to TF2.

### PR DESCRIPTION
**Pull request by stephenwithav**
As promised in #15.

`main.py` just needs a few tweaks.

### CODE REVIEW

**Comment by stephenwithav** [comment]
`main.py` and `losses.py` will be my goals for tomorrow and this weekend.

**Comment by xavysp** [comment]
> `main.py` and `losses.py` will be my goals for tomorrow and this weekend.

I have a question here [keras custom test step](https://stackoverflow.com/questions/61960251/how-to-set-custom-test-step-in-keras) maybe you can help me

Apparently,  when I use just use conv2d and rely (without batch normalization) it works but for efficiency I need it.


Cheers

**Comment by stephenwithav** [comment]
I will debug that tomorrow.  I took a coding break yesterday, so will finish the translation this afternoon.

Can you push the full code to a repo for me to clone?  Debugging Python is much easier for me, for now, that way.

**Comment by xavysp** [comment]
> I will debug that tomorrow. I took a coding break yesterday, so will finish the translation this afternoon.
> 
> Can you push the full code to a repo for me to clone? Debugging Python is much easier for me, for now, that way.

Don't worry maybe you are tired, so whenever you finish the Keras parts we ca see testing in DexiNed, have a good week.

Cheers,
